common/artifacts/000_wc_artifacts.txt

```diff
@@ -801,6 +1046,93 @@ dark_crystal = {
 
 	picture = "GFX_inv_misc_orb_03"
 }
+horn_of_cenarius = {
+	quality = 5
+	flags = { misc powerful_magical_artifact }
+
+	active = {
+		is_adult = yes
+		is_druid_class_trigger = yes
+	}
+	allowed_gift = {
+		always = yes
+	}
+
+	stacking = no
+	indestructible = yes
+
+	health = 0.75
+
+	monthly_character_prestige = 0.7
+	monthly_character_piety = 0.4
+
+	druidism_group_opinion = 5
+	dryad_group_opinion = 5
+	night_elf_group_opinion = 5
```

Comment:
no loc

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/67306481-b4e11b00-f4ff-11e9-86fd-0fb1cf696ab7.png)

</details>

Reply:
fixed